src/dotnet/APIView/ClientSPA/src/app/_services/code-panel/code-panel.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
```

Comment:
We should not need a service. We should just fetch the information from the parent component